src/resources/js/tickets.js

```diff
@@ -193,879 +132,962 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 		} );
 	}
 
-	$( document ).ready( function() {
-		if ( $event_pickers.length ) {
-			startofweek = $event_pickers.data( 'startofweek' );
+	/**
+	 * Returns the currently selected default ticketing provider.
+	 * Defaults to RSVP if something fails
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	function get_default_provider() {
+		var $checked_provider = $( 'input[name=default_ticket_provider]', '#tribe_panel_settings' ).filter( ':checked' );
+		return ( $checked_provider.length > 0 ) ? $checked_provider.val() : 'Tribe__Tickets__RSVP';
+	}
+
+	/**
+	 * Sets the ticket edit form provider to the currently selected default ticketing provider.
+	 * Defaults to RSVP if something fails
+	 *
+	 * @since TBD
+	 *
+	 * @param boolean force selection to RSVP
+	 * @return void
+	 */
+	function set_default_provider_radio( force_rsvp ) {
+		force_rsvp = undefined === force_rsvp ? false : true;
+		var $checked_provider = $( 'input[name="default_ticket_provider"]', '#tribe_panel_settings' ).filter( ':checked' );
+		var provider_id = 'Tribe__Tickets__RSVP_radio';
+
+		if ( ! force_rsvp && $checked_provider.length > 0 ) {
+			provider_id = $checked_provider.val() + '_radio';
+		 }
+
+		$( document.getElementById( provider_id ) ).prop( 'checked', true ).trigger('change');
+	}
+
+	/**
+	 * Returns the current global capacity (via the settings panel.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	function get_global_cap() {
+		var $global_capacity_edit = $( document.getElementById( 'settings_global_capacity_edit' ) )
+		return ( 0 < $global_capacity_edit.length && 0 < $global_capacity_edit.val() ) ? $global_capacity_edit.val() : 0;
+	}
+
+	/**
+	 * When a ticket type is edited we should (re-)establish the UI for showing
+	 * and hiding its history, if it has one.
+	 */
+	function show_hide_ticket_type_history() {
+		var $history = $tribe_tickets.find( '.ticket_advanced.history' );
+
+		if ( ! $history.length ) {
+			return;
 		}
 
-		var datepickerOpts = {
-			dateFormat     : date_format,
-			showAnim       : 'fadeIn',
-			changeMonth    : true,
-			changeYear     : true,
-			numberOfMonths : 3,
-			firstDay       : startofweek,
-			showButtonPanel: false,
-			onChange       : function() {
-			},
-			onSelect       : function( dateText, inst ) {
-				var the_date = $.datepicker.parseDate( 'yy-mm-dd', dateText );
-				if ( inst.id === 'ticket_start_date' ) {
-					$ticket_end_date.datepicker( 'option', 'minDate', the_date );
-				} else {
-					$ticket_start_date.datepicker( 'option', 'maxDate', the_date );
-				}
+		var $toggle_link      = $history.find( 'a.toggle-history' );
+		var $toggle_link_text = $toggle_link.find( 'span' );
+		var $history_list     = $history.find( 'ul' );
```

Comment:
Ditto here - seems like we define `$history_list` but don't use it after this point